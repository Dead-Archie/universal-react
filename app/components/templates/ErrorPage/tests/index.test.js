import { shallow } from 'enzyme';
import { fromJS } from 'immutable';
// import { ErrorPageDisconnected as ErrorPage } from '../ErrorPage';

const props = {
  errorData: fromJS({}),
};

xdescribe('ErrorPage Component', () => {
  test.skip('should render correctly', () => {
    const errorPage = shallow(<ErrorPage {...props} />);
    expect(errorPage).toMatchSnapshot();
  });
});
