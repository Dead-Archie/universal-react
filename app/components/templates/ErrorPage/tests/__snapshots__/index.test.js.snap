// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage Component should render correctly 1`] = `
<Layout
  className="row"
  hasRightGutter={false}
  id="content-wrapper"
  tabindex="-1"
  title="error"
>
  <HeadTag
    description="error page"
    title="error page"
  />
  <section
    id="notFoundContent"
    style={
      Object {
        "width": "100%",
      }
    }
  >
    Test Message
  </section>
</Layout>
`;
